README.md

```diff
@@ -86,6 +86,19 @@ You can enable preprocessing with the `preprocess` option:
 }
 ```
 
+You can use the `no-parser.js` entry as an ES module by enabling the `module` option:
```

Comment:
I can't say if this is a good solution, maybe @philpl has a better idea, but anyhow we should document this better since people won't have a clue what this means :laughing:

What about

> If you use Rollup or webpack v2 you can take advantage of tree-shaking by setting the `module` option to true.

Is that the case for the user? What other scenarios exist where somebody would want to use this?